### DISCUSSION TITLE
"delete" button at the end of the screen or repeat of all mass maintenance features/buttons

### DISCUSSION THREAD

**Discussion by kurti500** [jonaswinkler/paperless-ng#907] [Feature Requests]
Hi,
as I dump all files into paperless, I regularly need to mass-delete items I don't want to keep.
I go through the screen from top to button, highlighting the ones I want to delete. 
At the end of the page an additional "delete" button would be very handy, as otherwise you have to scroll all the page up and press delete there. 
Maybe repeating the whole "mass maintenance" buttons at the end of the page would be appreciated for other use-cases, too?

Great project by the way, really appreciate the hard work and dedication you guys put into this!

**Comment by anokfireball** [comment]
Instead of duplicating the buttons / interface, another option would be to have a sticky toolbar (i.e. it always is visible at the top even when scrolling). This would make it accessible from anywhere on a page without having to go all the way to the bottom or top.

**Reply by kurti500** [reply]
even better, I would say 😀
Tried to start simple with my request 😉

**Reply by jonaswinkler** [reply]
Hey @shamoon, is this something you'd like to look into? This sounds really useful.

**Reply by shamoon** [reply]
> Hey @shamoon, is this something you'd like to look into? This sounds really useful.

Yea let me play with it a bit, will report back!

**Reply by shamoon** [reply]
See PR #918
